apps/widget-configurator/vite.config.ts

```diff
@@ -1,9 +1,27 @@
 /// <reference types="vitest" />
-import react from '@vitejs/plugin-react'
+import react from '@vitejs/plugin-react-swc'
 import { defineConfig } from 'vite'
+import { nodePolyfills } from 'vite-plugin-node-polyfills'
 import viteTsConfigPaths from 'vite-tsconfig-paths'
 
+const plugins = [
```

Comment:
We don't use macros plugin in the code yet, afaik

Reply:
I'm mentioning because I had to add it https://github.com/cowprotocol/cowswap/pull/3258/files#diff-8564f4bf1a4e9e7f8b3b0c07f9d56587362b17cdd1e658450101fda069e3a0a7R19

And I got it from where it's used already https://github.com/search?q=repo%3Acowprotocol%2Fcowswap+macros&type=code

Reply:
No no, I mean we don't use it in `widget-configurator` app